scripts/prepare_machine-unknown-linux-musl.sh

```diff
@@ -1,47 +1,39 @@
-#!/usr/bin/env bash
+#!/usr/bin/env sh
 #
 # Copyright (c) Granulate. All rights reserved.
 # Licensed under the AGPL3 License. See LICENSE.md in the project root for license information.
 #
-set -euo pipefail
+set -eu
 
 # prepares the environment for building py-spy:
 # 1. installs the rust target $(uname -m)-unknown-linux-musl - this can build static binaries
-# 2. downloads, builds & installs libunwind with musl
 # 2. downloads, builds & installs libz with musl
 # I use musl because it builds statically. otherwise, we need to build with old glibc; I tried to
 # build on centos:7 but it caused some errors out-of-the-box (libunwind was built w/o -fPIC and rust tried
 # to build it as shared (?))
 # in any way, building it static solves all issues. and I find it better to use more recent versions of libraries
 # like libunwind/zlib.
 
-rustup target add "$(uname -m)"-unknown-linux-musl
+target="$(uname -m)"-unknown-linux-musl
+target_dir="/usr/local/musl/$target/"  # as searched for by the remoteprocess create.
+rustup target add "$target"
 
-apt-get update && apt-get install -y musl-dev musl-tools
+apk add --no-cache musl-dev make git curl  # git & curl used by next scripts
 
 mkdir builds && cd builds
 
-wget https://github.com/libunwind/libunwind/releases/download/v1.6.2/libunwind-1.6.2.tar.gz
-tar -xf libunwind-1.6.2.tar.gz
-pushd libunwind-1.6.2
-CC=musl-gcc ./configure --disable-minidebuginfo --enable-ptrace --disable-tests --disable-documentation
```

Comment:
Here we passed `--disable-minidebuginfo --enable-ptrace` but in the new build we no longer do that, is it necessary?

Reply:
Well, this is here from #68 . If I didn't write in comment then it's not important enough.